doc/ipython-notebooks/clustering/GMM.ipynb

```diff
@@ -11,7 +11,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "#### By Heiko Strathmann - <a href=\"mailto:heiko.strathmann@gmail.com\">heiko.strathmann@gmail.com</a> - <a href=\"github.com/karlnapf\">github.com/karlnapf</a> - <a href=\"herrstrathmann.de\">herrstrathmann.de</a>.   Based on the GMM framework of the <a href=\"https://www.google-melange.com/gsoc/project/google/gsoc2011/alesis_novik/11001\">Google summer of code 2011 project</a> of Alesis Novik - <a href=\"https://github.com/alesis\">https://github.com/alesis</a>"
+    "#### By Heiko Strathmann - <a href=\"mailto:heiko.strathmann@gmail.com\">heiko.strathmann@gmail.com</a> - <a href=\"http://github.com/karlnapf\">github.com/karlnapf</a> - <a href=\"http://herrstrathmann.de\">herrstrathmann.de</a>.   Based on the GMM framework of the <a href=\"https://www.google-melange.com/gsoc/project/google/gsoc2011/alesis_novik/11001\">Google summer of code 2011 project</a> of Alesis Novik - <a href=\"https://github.com/alesis\">https://github.com/alesis</a>"
```

Comment:
Now that we are here.
Can you split every sentence you touch in the notebook into separate lines? The diff is hard to read otherwise.
Just put a new line after every period.